doc/cdrwtool.1

```diff
@@ -135,11 +135,29 @@ variable and fixed packet sizes respectively.
 .IP "\fB\-o \fIoffset\fP"
 Set write offset.
 
+.SH BUGS
+Many modern drives refuse on the preparations to format new,
+blanked, or appendable CD-RW media. This causes a message like
+.nf
+  "Command failed: 55 ... - sense ..."
+.fi
+.br
```

Comment:
I would suggest to put an empty lines between text and error message. It is better readable. It can be done by starting a new paragraph by `.PP` macro. Error message can be in indented paragraph (`.IP` macro).

```suggestion
.IP
.B Command failed: 55 ... - sense ...
.PP
```